gcloud/datastore/batch.py

```diff
@@ -21,7 +21,7 @@ class Local(object):
 
 from gcloud.datastore import _implicit_environ
 from gcloud.datastore import helpers
-from gcloud.datastore import datastore_v1_pb2 as datastore_pb
+from gcloud.datastore import _datastore_v1_pb2 as datastore_pb
```

Comment:
@jgeewax How do you mean the way we currently document it? We don't document it anywhere.

Current the module is public, but this change makes it private, which means we don't guarantee the interface to users.

@tseaver Whether we make it public or not, people can always re-generate this file by using the `protoc` compiler with the [spec][1]. By making the module private, we are just saying we don't guarantee the interface (which we don't).

[1]: https://github.com/GoogleCloudPlatform/google-cloud-datastore/blob/21e029b2657ebb15fef5429c90c4315326855fd1/proto/datastore_v1.proto